codecov.yml

```diff
@@ -14,7 +14,7 @@ coverage:
         target: 94.0
         threshold: 0.1
       fully_covered:
-        target: 100.0
+        target: 99.95
```

Comment:
This section is for files/dirs that are fully covered. So the `fully_covered` is now lying a bit. I don't care that much but it is similarly problematic than my approach of marking the individual lines as uncoverable.

@okurz How would you handle it? One idea is to rather move the now problematic files/dirs out of there. It has the disadvantage of either moving the whole `t/` directory out or of having to specify tests individually.

Reply:
I don't see this change as acceptable for the reasons you stated. And we should really aim for 100% coverage. If that means mocking old/new qemu then do that. I am also ok to use "uncoverable" if there is a good reason and according comment.